samples/sensor/wsen_hids_2525020210002/README.rst

```diff
@@ -0,0 +1,40 @@
+.. _wsen-hids-2525020210002:
+
+WSEN-HIDS-2525020210002: Humidity sensor
+########################################
+
+Overview
+********
+ This sample periodically measures humidity and ambient temperature for
+ 50 sec in the interval of 500msec in polling mode. 
+ The result is displayed on the console.
+
+Requirements
+************
+
+ This sample uses the WSEN-HIDS sensor controlled using the I2C interface.
```

Comment:
We're trying to stop introducing new driver-specific samples and move toward having more generic samples for each sensor type. Please look at `samples/sensor/accel_polling` for example

Reply:
The samples for the drivers has been supplied because some of the drivers implement custom attribues or triggers. which without looking at the example would be really hard for the user to use.

Reply:
Can we postpone the new samples to another PR?

Reply:
i just removed them from this PR and will create a seperate one affter this one is merged.